### PR TITLE
[Merge] Development -> Main

### PR DESCRIPTION
**Pull request by Lord-Nickelbottoms**
Integrating chatbot logic and styling into Main

### CODE REVIEW

**Review by SilindileShabangu — APPROVED** [review]

File: scripts/chatbot.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const interactions = [
     { question: "What are you?", answer: "Experiment 626."},
     { question: "What is your purpose?", answer: "ʻOhana" },
     { question: "Status Error", answer: "System logs recorded." },
+    { question: "The Emperor requests your services.", answer: "Royal members pay a higher fee."},
     { question: "We've got a job to do.", answer: "😉"}
 ]
```
Comment (Sisamkele022) [inline scripts/chatbot.js:13]: This change adds a humorous, exclusive tone and aligns well with the existing personality and style.

**Review by Sisamkele022 — APPROVED** [review]

File: scripts/chatbot.js [hunk 2]
```diff
@@ -116,8 +117,7 @@ async function newChatbotMessage(chatbotResponse) {
 }
 
 function routeToContact() {
-    const contactElement = document.getElementById("socials-section")
-    contactElement.scrollIntoView( { behavior: "smooth", block: "start", inline: "center" } )
+    window.location.href = "contact.html"
```
Comment (Sisamkele022) [inline scripts/chatbot.js:120]: Why did you change the original smooth scrolling implementation cause I believe it was better as it allowed users to quickly navigate to the specific part of the contact page?

Comment (Lord-Nickelbottoms) [reply scripts/chatbot.js:120]: That implementation scrolled downwards to the footer for contacting me. This new implementation is meant to route to the contact page instead

**Comment by LefaJele-Masemola** [comment]
I love the idea of the chatbot's personality.

**Review by LefaJele-Masemola — APPROVED** [review]

**Review by Ayanda-Radebe — APPROVED** [review]
code is clean!
